rules/scripts/bitstreams_workspace.py

```diff
@@ -279,30 +351,34 @@ def _ConstructBazelString(self, build_file: Path, key: str) -> str:
         if key == 'latest':
             key = self.available['latest']
 
-        files_by_extension = self.GetFromCache(key)
-
-        if len(files_by_extension.get('orig', set())) != 1 or \
-           len(files_by_extension.get('splice', set())) != 1:
-            error_msg_lines = [
-                "Could not find the bitstreams to generate a BUILD file:" +
-                repr(build_file),
-                "in files_by_extension:" + repr(files_by_extension),
-                "using key:" + repr(key),
-            ]
-            logging.error('\n'.join(error_msg_lines))
-            sys.exit(1)
-
-        (test_rom_file, ) = files_by_extension['orig']
-        (rom_file, ) = files_by_extension['splice']
-
-        def filegroup_lines(name, src):
-            return [
-                'filegroup(',
-                '    name = "{}",'.format(name),
-                '    srcs = ["{}"],'.format(src),
-                ')',
-                '',
-            ]
+        manifest = self.GetFromCache(key)
+        designs = dict()
```

Comment:
nit, you could do:
```suggestion
        designs = collections.defaultdict(dict)
```

then you could remove the lines:
```python
if design_name not in designs:
    designs[design_name] = dict()
```
below.

Reply:
Thanks!